### ISSUE TITLE
Cannot login to manager

### ISSUE THREAD

**Issue by AdamJachocki** [PiranhaCMS/piranha.core#1542]
Hi, this is similar to: https://github.com/PiranhaCMS/piranha.core/issues/498
I am trying to use Piranha 8.4, because I work on .NetCore 3.1 For some reason, yesterday I was able to login into Piranha Manager and do some work. But today I am not able to login. I tried different browsers, I tried clearing cookies and cache... There is no security problem. I do not get any login error, I am just redirected to login page all the time. So, whenever I login, I get the login page again. Please help. What can I do more?

**Comment by tidyui** [comment]
Did you change roles/claims assignments on your user? Are you sure the user has access to the claim `Admin` which is needed to login?

**Comment by AdamJachocki** [comment]
I did not change anything. Just created user with SysAdmin role. Yesterday I was able to login. At this stage I closed the computer. Today I am not able to login. No changes were made.

**Comment by tidyui** [comment]
Like in #498, have you tried to login using incognito/private mode in your browser to make sure there's no cookies interfering with the login?

**Comment by AdamJachocki** [comment]
Yes, nothing worked

**Comment by jensbrak** [comment]
Is this running debug from VS or is it when deployed? 
Do you get any errors in the log that tells what is failing?

I had issues with the manager when deployed where I got 502 errors.
From one version of Piranha to the next it stopped working. While the cause of the errors was originating from Piranha, the actual error had nothing to do with Piranha, it was my reverse proxy that couldn't deal with an increase in header sizes.

I've also had issues with the Manager related to other similar errors (for instance too large cookies). And at some point there was an odd issue with the Manager resulting in not being able to login but without any errors at all.

**Comment by tidyui** [comment]
@AdamJachocki As I know from another issue that you're integrating Piranha into an already existing application, do you already have Identity set up in your application, or some other form of authentication? Just in case there might be a conflict.

**Comment by AdamJachocki** [comment]
Hi, 
- this is in my development environment on localhost (debug configuration)
- there are no error in the output pane
- I use identity in this project, but all authorization is in done by other one (WebAPI)
- on Chrome I can see in headers that there is 302 response
- ocassionaly it works; at the end of the day I was even able to create my custom page in manager
- I use custom domain -> which works fine in my project

I've just check it and today it also works. I have some idea that this could have something in common with git version control. When I gave up today, I changed my branch. But after some time, changed it again to branch with Piranha. And it started to work :|
